src/Symfony/Component/ErrorHandler/ErrorHandler.php

```diff
@@ -153,6 +153,32 @@ public static function register(self $handler = null, bool $replace = true): sel
         return $handler;
     }
 
+    /**
+     * Calls a function and turns any PHP warnings into \ErrorException.
+     *
+     * @return mixed Returns $function(...$arguments)
```

Comment:
@return mixed returns

Reply:
we use an uppercase letter quite often here
maybe `@return mixed What $function(...$arguments) returns`?